data/mods/gen8bdsp/formats-data.ts

```diff
@@ -1739,7 +1739,7 @@ export const FormatsData: {[k: string]: ModdedSpeciesFormatsData} = {
 	empoleon: {
 		tier: "UU",
 		doublesTier: "DUU",
-		randomBattleMoves: ["agility", "defog", "flashcannon", "icebeam", "scald", "stealthrock"],
+		randomBattleMoves: ["agility", "defog", "grassknot", "icebeam", "scald", "stealthrock"],
```

Comment:
Why remove Flash Cannon? It learns it by TM. Is Steel STAB just useless?

Reply:
Ice/Steel/Water can't hit water and steel doesn't provide anything over water stab or ice/grass coverage

Reply:
Especially since there are close to no fairy mons in the format